docs/sphinx/contribution/code-style-python.rst

```diff
@@ -0,0 +1,193 @@
+Python Coding Style Guide
+=========================
+
+CHIPSEC mostly follows the PEP8 with some exceptions.  This attempts to highlight those as well as clarify others.
+
+Consistency and readability are the goal but not at the expense of readability or functionality.
+
+If in doubt, follow the existing code style and formatting.
+
+
+#. PEP8
+
+    PEP 8 is a set of recommended code style guidelines (conventions) for Python.
+
+    `PEP 8 <https://www.python.org/dev/peps/pep-0008/>`_
+
+
+#. Linting tools
+
+    CHIPSEC includes a Flake8 configuration file
+
+    `CHIPSEC flake8 config <https://github.com/chipsec/chipsec/blob/master/.flake8>`_
+
+
+#. Zen of Python
+
+    Great philosophy around Python building principles.
+
+    `PEP 20 <https://www.python.org/dev/peps/pep-0020/>`_
+
+
+#. Headers and Comments
+
+    Use single line comments, a single hash/number sign/octothorpe '#'.
+
+    Should contain a space immediately after the '#'.
+
+    .. code-block:: python
+        # Good header comment
```

Comment:
May need a space between code block directive and comment

Reply:
Thanks! They should all be resolved.